src/PrestaShopBundle/Resources/views/Admin/Common/Grid/Columns/Content/bulk_action.html.twig

```diff
@@ -31,6 +31,15 @@
            name="{{ grid.id~'_'~column.id }}[]"
            value="{{ record[column.options.bulk_field] }}"
     >
+
+    {% for extra_field in column.options.extra_fields %}
+      <input type="checkbox"
+             title="{{ column.name }}"
+             class="js-bulk-action-checkbox"
+             name="{{ grid.id~'_'~column.id~'_'~extra_field }}[]"
+             value="{{ record[extra_field] }}"
```

Comment:
Damn, that indent looks weird :/

But I see that it looks the same upper... so no worries

Reply:
Functionality was removed entirely.